### ISSUE TITLE
xmonad contrib library

### ISSUE THREAD

**Issue by ghost** [lynnard/VIMonad#9]
As far as I can tell, this code is not in the contrib library. It seems like the natural fit.

**Comment by lingnand** [comment]
I agree - a big part of the code base was put together when I was hacking around and learning about the language - and some of the interesting features led to actually quite a few (arguably big) changes in other Contrib modules. Hence the reason for 'fork' rather than 'pull request'.

A long-standing task on my list is to go back and properly organize the code and merge it with the upstream Contrib; but unfortunately I've been constantly occupied in other things and still haven't got a chance for this.
